recipes/xarrayutils/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
It's fine to manually package the license file, but ideally it would be included in the `MANIFEST.in` upstream and just come in the source distribution. Since you're the packages maintainer, please consider doing that for future releases and then deleting this local copy.

Reply:
Ok thanks for the tip. Can I just do that in a future release? Or should I implement that now?

Reply:
A future release would be fine

Reply:
Awesome. Thanks. I have already added it in [xarrayutils/#57](https://github.com/jbusecke/xarrayutils/pull/57). Many thanks for your input.